lib/encryption/utils/bootstrap.dart

```diff
@@ -581,14 +581,6 @@ class Bootstrap {
       );
       Logs().v('Store the secret...');
       await newSsssKey?.store(megolmKey, base64.encode(privKey));
```

Comment:
can you also update the other places?

Reply:
This was the only place with `store` where we had this (broken) logic, so don't need anything more afaik

Reply:
That's correct

Reply:
Then how is the logic in https://github.com/famedly/matrix-dart-sdk/blob/200c9a5ad23bdae4d3446fe19ed36f7262db4283/lib/encryption/utils/bootstrap.dart#L493 different?